project.clj

```diff
@@ -9,6 +9,8 @@
                  [com.google.cloud/google-cloud-core "1.87.0"]
                  [com.google.cloud/google-cloud-logging "1.87.0"] ; do not exclude io.grpc/grpc-core io.grpc/grpc-api io.grpc/grpc-netty-shaded or else logging will fail to load classes
                  [com.google.cloud/google-cloud-storage "1.87.0"]
+                 [com.google.apis/google-api-services-compute "v1-rev214-1.25.0"]
+                 ;; [com.google.api-client/google-api-client "1.30.4"]
```

Comment:
While each service has its own API (https://cloud.google.com/apis/), it's weird that compute has a different version number pattern. How are developers expected to update them together in sync?

Just a thought: Would this [jclouds](https://cloud.google.com/compute/docs/api/libraries#jclouds) library add much benefit? "The [jclouds](http://jclouds.apache.org/) cloud API has support for Compute Engine and allows users to manage resources such as virtual machines, disks, and networking." It'd have to have big benefits to be worth the risks of lagging behind and additional bugs.

Should we update the other libs to `1.99.0`?

Reply:
> How are developers expected to update them together in sync?

No clue on this one, I'm not sure how much the different teams coordinate based on this. I see no evidence anyone is releasing things in sync.

> Would this jclouds library add much benefit?

I checked out `jclouds`.... my issue is that it adds another layer of where things could go wrong, and it is trying to generalize over several cloud providers, which means there will be compromises for any particular provider. I'm happy just doing the direct thing until we have another reason. 

> Should we update the other libs to 1.99.0?

We can! I'll take a look.

Reply:
Upgrading to 1.99.0 failed for `google-cloud-core`

Reply:
Cool. And thankfully, the gaia project doesn't have its own dependencies on these same libs.